website/docs/r/private_key.html.md

```diff
@@ -24,43 +23,71 @@ state and does not create any external managed resources.
 ## Example Usage
 
 ```hcl
+// ECDSA key with P384 elliptic curve
 resource "tls_private_key" "example" {
   algorithm   = "ECDSA"
   ecdsa_curve = "P384"
 }
+
+// RSA key of size 4096 bits
+resource "tls_private_key" "example" {
```

Comment:
Having multiple resources with the same name will generate an error if this whole configuration happens to be copy-pasted -- rather than using comments, the resource name labels could be adjusted to denote the different algorithms

```suggestion
resource "tls_private_key" "rsa-4096-example" {
```

Reply:
Silly me, of course! Will fix.